tools/build/binary-release/Linux/scripts/set-env.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+# Sourced from https://stackoverflow.com/a/29835459/1975049
+rreadlink() (
+  target=$1 fname= targetDir= CDPATH=
+  { \unalias command; \unset -f command; } >/dev/null 2>&1
+  [ -n "$ZSH_VERSION" ] && options[POSIX_BUILTINS]=on
+  while :; do
+      [ -L "$target" ] || [ -e "$target" ] || { command printf '%s\n' "ERROR: '$target' does not exist." >&2; return 1; }
+      command cd "$(command dirname -- "$target")"
```

Comment:
[Bash pitfall #14](https://mywiki.wooledge.org/BashPitfalls#cd_.2Ffoo.3B_bar)? `cd` used without checking if it succeeded.

Reply:
Done.